lib/azuremetadata/azuremetadatautils.py

```diff
@@ -63,7 +64,13 @@ def _parse_data(self, data, parent_key=''):
     def print_help(self):
         self._pretty_print(self.PRINT_MODE_HELP, self._data)
 
-    def print_pretty(self, print_xml=False, print_json=False, data=None, file=None):
+    def print_api_versions(self):
+        self._pretty_print(self.PRINT_MODE_APIS, self._data, 1)
```

Comment:
Instead of passing this list into `_pretty_print` and introducing `PRINT_MODE_APIS` -- why not just iterate over the list and `print()` each version? There are no nested structures inside, indentation is the same.

Alternatively, use `_pretty_print` to print versions in plain text, XML or JSON (i.e., don't introduce `PRINT_MODE_APIS`).

Reply:
Will do. I thought to make use of the printing methods already in place in utils.

Reply:
Done.

Reply:
:+1: 

The printing methods are for printing nested data structures recursively in different output formats. It takes just two lines of code to print the API versions -- it just seems cleaner.